config/tsconfig.integration.json

```diff
@@ -0,0 +1,7 @@
+{
+  "extends": "./tsconfig.base.json",
+  "compilerOptions": {
+    "moduleResolution": "Bundler",
```

Comment:
Set to bunlder according to: https://www.totaltypescript.com/tsconfig-cheat-sheet